CodeConverter/CSharp/CommonConversions.cs

```diff
@@ -714,4 +709,17 @@ public async Task<string> GetClassificationLastTokenAsync(VBSyntax.SimpleImports
 
         return spans.Last().ClassificationType;
     }
+
+    public RefKind GetCsRefKind(IParameterSymbol vbParameter, Microsoft.CodeAnalysis.VisualBasic.Syntax.ParameterSyntax optionalParameterSyntax = null)
+    {
+        if (this.GetCsOriginalSymbolOrNull(vbParameter) is IParameterSymbol csParam) {
```

Comment:
This means it'll probably work when there's no syntax available too. See note in https://github.com/icsharpcode/CodeConverter/blob/master/.github/CONTRIBUTING.md#moving-away-from-legacy-patterns-in-the-code